### DISCUSSION TITLE
Better support for front matter

### DISCUSSION THREAD

**Discussion by peterlerche** [shuzijun/markdown-editor#33] [Ideas]
Would it be possible to support TOML (https://toml.io/en/) format in the front matter section of markdown files.

Your plugin is a game changer - thank you

**Comment by shuzijun** [comment]
I don't quite understand the usage of this function, can you describe the function in detail?

**Comment by peterlerche** [comment]
I see, I was more than wage, Sorry. 

Visually, YAML looks so much better than TOML in your editor. 
When displaying YAML the front matter section is nicely colored and thereby visually separated from the markdown content. 
Using TOML "+++" does not enable this visual separation.  
I know that it is only a minor issue but when the front matter TOML section is large is does make it harder to separate the two sections. 

And thanks for a really good plugin.....
